components/board/jetson/data.go

```diff
@@ -361,7 +361,7 @@ var boardInfoMappings = map[string]commonsysfs.BoardInformation{
 		},
 	},
 	jetsonOrin: {
-		[]commonsysfs.PinDefinition{
+		[]genericlinux.PinDefinition{
```

Comment:
I didn't validate this definition, nor do I even know how to :).
Do you need these pin mappings reviewed or are these confirmed to be accurate?

Reply:
#1623 says it comes from https://github.com/NVIDIA/jetson-gpio/blob/master/lib/python/Jetson/GPIO/gpio_pin_data.py#L52, and it does seem consistent with what I've seen from the Jetson Orin and Jetson Nano documentation. I haven't reviewed each pin in detail (especially for the boards whose hardware I don't have!), but I trust Danenberg to have copied and pasted correctly. :upside_down_face:

Reply:
Let's validate the orin one as that's the one we want to perfect now.

Reply:
Yes, the ones for the Orin are the same as that .py file. Since then, NVidia has added separate pin definitions for the Orin NX (the mid-level board that was due out in January and is officially released but cannot be purchased because all sellers are currently out of stock), but I think that should be added in a separate PR.